### ISSUE TITLE
optional notification sounds for traffic cams instead of TTS

### ISSUE THREAD

**Issue by Schweber** [organicmaps/organicmaps#8941]
⚠ Have you searched for similar, already existing issues?
yes

**Is your feature request related to a problem? Please describe.**
I don't have a TTS engine on my phone and am thus not getting any audible warning for traffic cams any more. In older versions there was a notification sound if i remember correctly.

**Describe the ideal solution**
Reintroduce notification sounds for traffic cams so that the user can decide, if he wants a sound and/or a TTS notification.

**Describe alternatives you have considered**
The availability of a TTS engine can't be taken for granted on custom ROMs. Thus i would like `organic maps` to consider this and provide alternative options.

**Comment by RedAuburn** [comment]
make sure "speed camera warnings" are enabled in the "Voice Instructions" tab in the settings

**Comment by ales-tsurko** [comment]
I'd second this. It would be useful for vehicles without google services (without TTS). Particularly I'd like to see this feature for all things, which requires attention (crossing approaching, route changes, speed cams, etc.). Just a simple alert would do the trick.

**Comment by biodranik** [comment]
Can you please confirm that installing TTS engine fixes the camera beep warning?
See https://organicmaps.app/faq/voice/text-to-speech-android-tts/ (SherpaTTS should be good enough).

**Comment by Schweber** [comment]
I can't get TTS to work on my device. That's why it would be great if there was the option of just using sounds and no TTS.

**Comment by biodranik** [comment]
So you tried to install different TTS engines from the link above (including recently added SherpaTTS) and they all didn't work? Which custom ROM is it? Do you know why TTS is broken there?

**Comment by Schweber** [comment]
I got Sherpa TTS working now (i had to reboot for it to be recognised).

TTS in organicmaps is working now and i'm getting TTS warnings about speedcams. However, i get no warning sounds or anything like that and i'm only warned about speedcams if i have verbal turn-indications (i don't know the english term) enabled.

In summer 2022, organicmaps warned me with a warning sound without a TTS engine installed and without having verbal turn-indications enabled. I would like to have this capability back, please.

**Comment by biodranik** [comment]
AFAIR, we did not refactor the TTS engine since the beginning of OM.

**Comment by zloidemon** [comment]
> beginning of OM in 2021

[Alexander Borsuk](https://github.com/biodranik) The project was initiated in December 2020 prior to your involvement. Your contributions began later and were distinct from the project's foundational setup.

**Comment by biodranik** [comment]
@zloidemon thanks for the spam and for spreading lies.

I registered organicmaps organization at GitHub on December 21, 2020, where Viktor @vng and I started development on the project. Only a week after that, omaps organization was registered by Roman. Later, we agreed to continue development in Roman's omaps org and renamed it. And Roman deleted our initial org.

Proof from GitHub support:

<img width="1003" height="822" alt="Image" src="https://github.com/user-attachments/assets/b17892bf-49c4-4f25-acbb-77248b8a0b8e" />

Any further spam from you will lead to a permanent ban.

**Comment by zloidemon** [comment]
> Any further spam from you will lead to a permanent ban.

[Alexander Borsuk](https://github.com/biodranik) Could you kindly share any community-approved documentation that formally defines what constitutes 'spam' within the project?

I kindly request that [Alexander Borsuk](https://github.com/biodranik) provide any legal documentation or contractual provisions authorizing the imposition of bans or other disciplinary measures against the project’s founders/members. Please clarify the basis for such authority at your earliest convenience.

> I registered organicmaps organization at GitHub on December 21, 2020, where Viktor [@vng](https://github.com/vng) and I started development on the project. Only a week after that, omaps organization was registered by Roman. Later, we agreed to continue development in Roman's omaps org and renamed it. And Roman deleted our initial org.

Before proceeding with further analysis of these claims, I would strongly encourage reviewing the existing publications documenting the project’s history.

1. [[docs] Factual history of the project](https://github.com/organicmaps/organicmaps/pull/10640)
2. [Organic Maps Turns 4: The Privacy-Focused Alternative to Google Maps](https://organicmaps.app/news/2024-12-20/organicmaps-turns-four/)

The activity of [Alexander Borsuk](https://github.com/biodranik) is attempting to engage in is commonly referred to as **cybersquatting**, which may constitute a violation of DCO & COC. Furthermore, the organicmaps domain in question was registered prior to your registration of maps.me, meaning you were not the first to claim it. 

<img width="850" height="285" alt="Image" src="https://github.com/user-attachments/assets/65d43160-e634-4661-94db-1d17dddce6e8" />


To ensure an accurate historical record, it would be valuable to verify the timelines you've provided regarding the project's development.


1. It appears [Alexander Borsuk](https://github.com/biodranik) is asserting that he was the first to create the fork. However, based on available records, the original repository was likely a blank template. This suggests it may have served as an organizational template rather than a derivative work, given the absence of modifications attributable to his at that stage.

```
> git log --since="2020-12-21" --until="2020-12-27" --format='%aN' |sort -u
Alexey Osminin
Arsentiy Milchakov
Maxim Pimenov
Olga Khlopkova
Tatiana-yan
```

2. According to documented activity, the first public beta version complete with generated maps and all necessary features was deployed for testing on **January 4, 2021**. To better understand the development process, it may be useful to examine the Git commit history, as this would provide insight into the actual contributions and workload during the beta phase. Would [Alexander Borsuk](https://github.com/biodranik) be able to share specific commit logs or collaborative records from that period? 

```
> git log --since="2020-12-27" --until="2021-01-04" --format='%aN' |sort -u
Maksim Andrianov
Roman Tsisyk
tatiana-yan
Veniamin Gvozdikov
```

<img width="887" height="393" alt="Image" src="https://github.com/user-attachments/assets/671affc1-9378-4fd9-aa5d-8f3e486a09d3" />

3. Publicly accessible records indicate that both [Alexander Borsuk](https://github.com/biodranik) and @vng  were formally added as contributors approximately **51 days after** the initial release of the application and maps. This can be verified through contributor activity logs and profile histories. Would [Alexander Borsuk](https://github.com/biodranik) be able to provide additional context or documentation regarding your involvement prior to this date?

<img width="778" height="262" alt="Image" src="https://github.com/user-attachments/assets/e7217df8-d089-4122-9ef6-8a96fd26e150" />

<img width="789" height="253" alt="Image" src="https://github.com/user-attachments/assets/b36338d5-df07-4942-a4c7-bf2147e0441c" />

4. An interesting chronological discrepancy arises when examining the project’s early development. According to contribution records, @ooprizrakoo involvement predates [Alexander Borsuk](https://github.com/biodranik) formal association with the project.

<img width="840" height="296" alt="Image" src="https://github.com/user-attachments/assets/01c3fdef-6489-45a3-914a-7526843bc0ae" />

5. The rebranding from OMaps to OrganicMaps raises some historical questions regarding project continuity and attribution. While you now claim founder status and creation of the domain, documented contributions show that neither [Alexander Borsuk](https://github.com/biodranik) nor Viktor were involved in the initial development phase. While the original OMaps project already had a functional infrastructure and public.

6. Regarding the rebranding decision from OMaps to OrganicMaps, it's important to note that this change was democratically approved by the project's founding team. Historical records show that at least two of the three original OMaps founders - specifically @ooprizrakoo and @rtsisyk - formally voted in favor of adopting the OrganicMaps name. This documented consensus among the project's creators lends institutional legitimacy to the name change and subsequent domain transition.

<img width="882" height="520" alt="Image" src="https://github.com/user-attachments/assets/a75d8d16-0109-45f6-8bb9-647241f1d729" />

**Comment by Schweber** [comment]
> Can you please confirm that installing TTS engine fixes the camera beep warning?

Are you getting warning sounds on your device?

**Comment by biodranik** [comment]
Yes, if TTS was set up correctly (restarting OM is recommended after changing TTS settings), note that the default setting is "warn if speeding".

**Comment by Schweber** [comment]
hmm, i have it set to "always warn" but it never does
